### DISCUSSION TITLE
Icon and Name of App

### DISCUSSION THREAD

**Discussion by dinhthengu** [ReVanced/revanced-discussions#1102] [Question] [answered]
Hi admin. Can I ask if there is a way to change the app icon and app name like the youtube app (like in version 19.34.42 in Android). Thank you very much admin.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
Include the custom branding patch if you want the ReVanced app icon.

If you want a custom icon then use the patch option in custom branding.

**Reply by dinhthengu** [reply]
Many thanks bro. I will try it. But iam not good with IT 😥😥😥

**Reply by dinhthengu** [reply]
> Include the custom branding patch if you want the ReVanced app icon.
> 
> If you want a custom icon then use the patch option in custom branding.

Hi bro. anyway, in the next update, I hope the admin adds video quality icon and video repeat to the video player like the picture, it would be great
![IMG_1732801783356_1732801823489](https://github.com/user-attachments/assets/832ec9bf-999a-4edc-95a4-4285ad5f4916)
